signin/signin.js

```diff
@@ -0,0 +1,64 @@
+const emailInput = document.querySelector('#email')
+const errorM = document.querySelector('.errorM')
+const pwInput = document.querySelector('#password')
+const errorP = document.querySelector('.errorP')
+const button = document.querySelector('.btn')
+const form = document.querySelector('form')
+const eye = document.querySelector('.eye')
+const inputs = document.querySelectorAll('.signInput')
+
+// 입력해주세요
+function emailError(e) {
+  console.log('focusout')
+  // 왜 e.target.value=''하면 적용이 안되는지ㅠ
+  if(!e.target.value.trim()) {
+    errorM.textContent = '이메일을 입력해 주세요.'
+  } else{
+    errorM.textContent=''
+  }
+}
+function pwError(e) {
+  if(!e.target.value.trim()) {
+    errorP.textContent = '비밀번호를 입력해 주세요.'
+  } else{
+    errorP.textContent=''
+  }
+}
+
+// 올바른 이메일 주소가 아닙니다
+function emailCheck(e) {
+  let e_style = /([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/
```

Comment:
정규표현식은 변하는 값이 아니기에 const 로 선언해주셔요!
```suggestion
  const e_style = /([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/
```